sound/soc/sof/intel/hda-loader.c

```diff
@@ -373,26 +376,147 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 	return ret;
 }
 
+/* Code loader helper APIs */
+static int cl_skl_cldma_setup_bdle(struct snd_sof_dev *sdev,
```

Comment:
We already have a function for creating BDLE and should reuse it.

Reply:
OK, I will check if it can be reused.